schemas/stsci.edu/asdf/0.1.0/core/history_entry.yaml

```diff
@@ -0,0 +1,28 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/0.1.0/core/history_entry"
+title: |
+  An entry in the file history.
+
+tag: "tag:stsci.edu:asdf/0.1.0/core/asdf"
+type: object
+properties:
+  description:
+    description: |
+      A description of the transformation performed.
+    type: string
+
+  time:
+    description: |
+      A timestamp for the operation.
+    type: string
+    format: date-time
+
+  software:
+    description: |
+      A description of the software that performed the operation.
```

Comment:
I wouldn't be opposed to changing it something like
```
oneOf:
    - $ref: software
    - type: array
      items: {$ref: software}
```
That's pretty harmless--if someone wanted to use multiple they could, but they don't have to either.